src/app/_components/Feedback/Feedback.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import LoveIcon from '@icons/loveIcon.svg';
+import { useCreateFeedback } from '@/app/_hooks';
+import SendFeedback from '@/lib/validationSchemas/sendFeedbackSchema';
+import { Modal } from '../Modal';
+import { TextInputField } from '../InputFields';
+import { CheckBox } from '../CheckBox';
+import { TextArea } from '../TextArea';
+import { PillButton } from '../PillButton';
+import { cn } from '@/utils/cn';
+import { buttonColorVariant, buttonVariant } from '../PillButton/style';
+
+const MAX_LENGTH = 320;
```

Comment:
I don't like having magic numbers in my code.

Reply:
Rather than just being some integer 320, assigning it to MAX_LENGTH variable describes what it actually means and will help you understand code better when you come back later.

Reply:
could not agree to that completely :)

seeing the code

```
...
maxLength={320}
...
```

means 'there is an max length argument, and im passing 320' - so we kinda can make assumption that 320 _is_ max length

in this code

```
...
maxLength={MAX_LENGTH}
...
```

we can take 'there is max length argument, and max length constant passed in' - where second part is kinda makes no sense, as it is not adding anything about 'what it is actually are', there are no additional sense in there

what could be done ( if still wanted ) is something like
```
...
maxLength={MAX_MESSAGE_LENGTH}
...
```
as this adds some bits of information why there is 320 - because it is a 'message' and business rules are that it has to be <=320